sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -92,3 +92,8 @@ select abs(-3.13), abs('-2.19');
 
 -- positive/negative
 select positive('-1.11'), positive(-1.11), negative('-1.11'), negative(-1.11);
+
+-- width_bucket
```

Comment:
Do we need end-to-end tests here? I think we already cover these cases in other test suites.